src/inc/jithelpers.h

```diff
@@ -403,6 +403,9 @@
 
 #endif // COR_JIT_EE_VERSION
 
+    JITHELPER1(CORINFO_HELP_JIT_PINVOKE_BEGIN, NULL, CORINFO_HELP_SIG_UNDEF, MDIL_HELP_UNDEF)
```

Comment:
The JIT-EE interface should be consistent accross runtimes. It means that each runtime may pay a tiny price for it by having unused bits here and there, but this price is worth paying to avoid ifdef hell.